### PR TITLE
[api][SIMS #645]Offering and calculations updated

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Changed the way that school amount and student amount are calculated;
- Consumed the new "years of study" from offerings.

### CODE REVIEW

**Review by abschwenker — APPROVED** [review]
lgtm

**Review by dheepak-aot — APPROVED** [review]
LGTM.

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/api/src/utilities/disbursements-utils.ts [hunk 1]
```diff
@@ -62,3 +60,28 @@ export function round(decimalValue: number | string): number | null {
...
 }
+
+/**
+ * Ensures that a number is rounded up to the decimal places
+ * and have the decimal part combined into the integer part.
+ * For instance:
+ * - 12345.78999 = 1234579 (79 is the decimal part).
+ * - 123.999 = 12400 (00 is the decimal part).
+ * - 123.9 = 12390 (90 is the decimal part).
+ * - 123 = 12300 (00 is the decimal part).
```
Comment (ann-aot) [inline sources/packages/api/src/utilities/disbursements-utils.ts:71]: thanks for the comment

**Review by ann-aot — APPROVED** [review]
LGTM 👍

**Pull request merged**